plugins/modules/consul_session.py

```diff
@@ -20,36 +20,30 @@
   - python-consul
   - requests
 author:
-  - Steve Gargan (@sgargan)
-extends_documentation_fragment:
```

Comment:
Please undo removing all documentation improvements from this module.

Also, better move changes in this module to a separate PR.

Reply:
Thanks for reviewing! Sure, I'll split this into a separate PR